docs/how_to/bucketing.md

```diff
@@ -45,9 +45,10 @@ To enable bucketing, MXNet needs to know how to construct a new unrolled symboli
 
 
 ```python
-model = mx.model.FeedForward(
-        ctx     = contexts,
-        symbol  = sym_gen)
+model = mx.mod.BucketingModule(
+        sym_gen             = sym_gen,
```

Comment:
there is no sym_gen argument.

Please actually run each example to make sure they work.